src/schema.js

```diff
@@ -0,0 +1,38 @@
+const { buildSchema } = require('graphql');
+
+const schema = buildSchema(`
+  type User {
+    userName: String!
+  }
+
+  type Sheets {
+    sheets: [Sheet]
+  }
+
+  type Sheet {
+    sheet: [Rows]
```

Comment:
I think this should read `rows: Rows`

Reply:
I did `sheet: [Rows]` because I was following the pattern in the github-api schema, which makes sense to me. The key is the same as the object name which points to the data within that object. Here is a snippet from the github-api:
```
  type Repos {
    repos: [Repo]
  }
```

Reply:
@meghanprestemon I see where you are coming from, and I think we can do what you are suggesting. I think however that we need to change the return type as I suggest, to return a single `Rows` which is itself a collection of `[Row]`.

```
type Sheet {
  sheet: Rows
}
```

Reply:
Okay - I thought it _had_ to be inside brackets. I'm sorry - I didn't understand your suggestion.

Done!